src/Castle.Windsor/MicroKernel/IKernelInternal.cs

```diff
@@ -51,7 +51,7 @@ public interface IKernelInternal : IKernel
 
 		IDisposable OptimizeDependencyResolution();
 
-		object Resolve(Type service, IDictionary arguments, IReleasePolicy policy);
+		object Resolve(Type service, IDictionary arguments, IReleasePolicy policy, bool ignoreParentContext = false);
```

Comment:
Adding this parameter is a breaking change to implementors of `IKernelInternal`. Given the name, I'm assuming users are expected to _not_ be implementing this interface.

Reply:
> I'm assuming users are expected to _not_ be implementing this interface.

Correct, it is there because Castle used to have two containers, the MicroKernel and Windsor, the two got merged but obviously still visible from the code base.